app/assets/javascripts/tabs.js

```diff
@@ -64,19 +66,35 @@
 
       })
 
+     Array.prototype.forEach.call(tabLinks, function (listItem) {
```

Comment:
This has the same issue as the other PR.
The elements of `tabLinks` are given a 'click' handler twice, once from line 59 and again from line 69.
The name of the parameter in the passed function (`tabLink``or `listItem`) may be different but it'll still be the same element from the `tabLinks`.